bin/src/MixinGenerator.php

```diff
@@ -0,0 +1,436 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Webmozart\Assert\Bin;
+
+use InvalidArgumentException;
+use ReflectionClass;
+use ReflectionException;
+use ReflectionMethod;
+use RuntimeException;
+use Webmozart\Assert\Assert;
+
+final class MixinGenerator
+{
+    /**
+     * @psalm-var list<string>
+     *
+     * @var string[]
+     */
+    private $unsupportedMethods = [
+        'nullOrNotInstanceOf',  // not supported by psalm (https://github.com/vimeo/psalm/issues/3443)
```

Comment:
https://github.com/vimeo/psalm/issues/3443 seems to be solved in upstream

Reply:
I think he only fixes parser to not crash, but the type would still not be supported.

Reply:
Its fine if its not going to be supported, as long as we keep the comment.